assets/css/packages/s11-floating-links.css

```diff
@@ -0,0 +1,197 @@
+.s11-floating-links,
+.s11-floating-links::before,
+.s11-floating-links::after,
+.s11-floating-links *,
+.s11-floating-links *::before,
+.s11-floating-links *::after {
+	box-sizing: border-box;
+}
+
+.s11-floating-links {
+	position: fixed;
+	right: 48px;
+	bottom: 48px;
+	z-index: 1000;
+	display: none;
+	flex-direction: column;
+	align-items: flex-end;
+	gap: 16px;
+}
+
+.s11-floating-links.s11-fadein,
+.s11-floating-links.s11-fading {
+	display: flex;
+}
+
+.s11-floating-links-logo-icon {
```

Comment:
The X and Percie aren't quite centered vertically in the circle. This css fixes it for me:
`.s11-svg-wrapper{display:flex}`